ansible_collections/arista/avd/plugins/plugin_utils/utils/is_new_ansible.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2025 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from ansible.release import __version__ as ansible_version
+
+IS_NEW_ANSIBLE = ansible_version.startswith(("2.2", "2.19"))
```

Comment:
I would change the name to `ANSIBLE_VERSION_ABOVE_2.19` as we may get some `IS_NEW_NEW_ANSIBLE` down the line otherwise

also this file should probably best be called `constants.py`